internal/querynode/tsafe_replica.go

```diff
@@ -114,15 +115,16 @@ func (t *tSafeReplica) removeTSafe(vChannel Channel) error {
 	if t.tSafes[vChannel].ref == 0 {
 		safer, err := t.getTSaferPrivate(vChannel)
```

Comment:
remove the err logic for getTSaferPrivate,we don't need to return err in removeTSafe, only return true or false

Reply:
fixed